third_party/opengl/include/EGL/eglplatform.h

```diff
@@ -95,20 +94,20 @@ typedef struct gbm_device  *EGLNativeDisplayType;
 typedef struct gbm_bo      *EGLNativePixmapType;
 typedef void               *EGLNativeWindowType;
 
-#elif defined(__ANDROID__)
+#elif defined(__ANDROID__) || defined(ANDROID)
```

Comment:
you need to remove `defined (ANDROID)` here, maybe as a separate commit.  See "gfxstream: workaround Soong glibc build" for the reason why